src/contract.js

```diff
@@ -36,6 +36,10 @@ export class Contract {
     return StrKey.encodeContract(this._id);
   }
 
+  toString() {
```

Comment:
what prompted the need to add this, should it have js-doc and be included in [Contract typedef](https://github.com/stellar/js-stellar-base/blob/better-contract-footprint/types/index.d.ts#L28) ?